.chloggen/metric-server-hostproc.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Sorry for my confusion :facepalm: since this is a small internal change I think we can drop the release note altogether. Apologies, I did not realize this was internal at first.

Reply:
OK, so I'll delete the changelog entry.